op-e2e/actions/interop/proofs_test.go

```diff
@@ -382,42 +380,20 @@ func TestInteropFaultProofs_Cycle(gt *testing.T) {
 	actEmitA := emitter.EmitMessage(alice, "hello")
 	actEmitB := emitter.EmitMessage(alice, "world")
 
-	// txToID creates the would-be message identifier of the next block
-	txToID := func(chain *dsl.Chain, tx *gethTypes.Transaction) inbox.Identifier {
-		status := chain.Sequencer.SyncStatus()
-		return inbox.Identifier{
-			Origin:      *tx.To(),
-			BlockNumber: big.NewInt(int64(status.UnsafeL2.Number + 1)),
-			LogIndex:    big.NewInt(0),
-			Timestamp:   big.NewInt(int64(status.UnsafeL2.Time + 2)),
-			ChainId:     chain.ChainID.ToBig(),
-		}
-	}
-	txToPayload := func(data []byte) []byte {
-		topic := crypto.Keccak256Hash([]byte("DataEmitted(bytes)"))
-		var msg []byte
-		msg = append(msg, topic.Bytes()...)
-		dataHash := crypto.Keccak256Hash(data)
-		msg = append(msg, dataHash.Bytes()...)
-		return msg
-	}
-
 	actors.ChainA.Sequencer.ActL2StartBlock(t)
 	actors.ChainB.Sequencer.ActL2StartBlock(t)
 
 	// create init messages
-	emitTxA, from := actEmitA(actors.ChainA)
-	require.NoError(t, actors.ChainA.SequencerEngine.EngineApi.IncludeTx(emitTxA, from))
-	emitTxB, from := actEmitB(actors.ChainB)
-	require.NoError(t, actors.ChainB.SequencerEngine.EngineApi.IncludeTx(emitTxB, from))
+	emitTxA := actEmitA(actors.ChainA)
+	emitTxA.Include(actors.ChainA.SequencerEngine.EngineApi)
```

Comment:
The `Include` method can implicitly use the `chain` in the `GeneratedTransaction`. So this would look like: `emitTxA.Include()`
Since the transaction was originally created for a particular chain, it'll be inappropriate to submit the tx elsewhere.